configs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@aragon/osx-commons-configs",
+  "author": "Aragon Association",
+  "version": "1.0.0",
+  "license": "MIT",
+  "typings": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "lint:ts": "cd .. && yarn run lint:configs:ts",
+    "prepare": "yarn run build",
+    "analyze": "size-limit --why",
+    "clean": "rm -Rf dist"
+  },
+  "peerDependencies": {},
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint:ts"
+    }
+  },
+  "devDependencies": {
+    "typescript": "^5.1"
```

Comment:
Should we use the same typescript everywhere?
![Bildschirmfoto 2023-12-20 um 15 05 26](https://github.com/aragon/osx-commons/assets/20623991/6fdc009e-714e-44c9-9b8c-13a92768fa68)

Reply:
updated it to be 5.2.2 but we cannot use everywhere the same version because of dependencies..